src/item/VanillaItems.php

```diff
@@ -644,4 +662,24 @@ private static function registerArmorItems() : void{
 		self::register("netherite_leggings", new Armor(new IID(Ids::NETHERITE_LEGGINGS), "Netherite Leggings", new ArmorTypeInfo(6, 556, ArmorInventory::SLOT_LEGS, 3, true, material: ArmorMaterials::NETHERITE()), [EnchantmentTags::LEGGINGS]));
 	}
 
+	private static function registerSmithingTemplates() : void{
+		self::register("netherite_upgrade_smithing_template", new Item(new ItemIdentifier(ItemTypeIds::NETHERITE_UPGRADE_SMITHING_TEMPLATE), "Smithing Template"));
```

Comment:
`
```suggestion
		self::register("netherite_upgrade_smithing_template", new Item(new IID(Ids::NETHERITE_UPGRADE_SMITHING_TEMPLATE), "Smithing Template"));
```
The imports are aliased for readability

Reply:
Also, why is this just called "smithing template"? Shouldn't it have a more specific name?